service/service.go

```diff
@@ -27,6 +27,9 @@ type Service struct {
 	// It represents the service uniquely.
 	Hash string `hash:"-"`
 
+	// HashVersion is the version of the algorithm used to calculate the of the hash
+	HashVersion string `hash:"-"`
```

Comment:
I don't think `hash:"-"` is used anymore but just to keep in consistent, let's pretend it is and in that case the version is required in the hashing